server/games/game.py

```diff
@@ -800,13 +802,19 @@ def to_dict(self):
             GameState.ENDED: 'closed',
             GameState.INITIALIZING: 'closed',
         }.get(self.state, 'closed')
+        game_type = {
+            GameType.COOP: 'coop',
+            GameType.CUSTOM: 'custom',
+            GameType.MATCHMAKER: 'matchmaker',
+        }.get(self.game_type, 'custom')
```

Comment:
Just make the enum values be strings and do `self.game_type.value`

Reply:
Or maybe make this a method of `GameType`, we might need the integer values for hooking it up to the database later.

Reply:
I copied the method of gameState, i'll make it more nice and neat.